app/javascript/components/CreateSurvey.js

```diff
@@ -209,6 +210,11 @@ const CreatSurvey = () => {
                                         <DeleteIcon/></Button>
                                     </Stack>
                                     )
+                                case questionType.MULTIPLE_CHOICE:
+                                    return(
+                                        //Need to be replaced with survey id
+                                        <McQuestion survey_id={1}/>
```

Comment:
this should be replaced

Reply:
I believe what should happen is when you first load this page you should call the API to create a new survey with default values (Title = New Title and isLive = false) (this is described in the tech doc btw!). I have very little experience with react and frontend development, but I think that can be done in a componentDidMount() method. Then take the response of that and save the survey id, title, isLive values into the props/state of this file and then on this line, you can make use of the props or state

Reply:
Yeah, I don't know if the create survey is working properly atm, so this is more of a stub for my own testing.